stabilizer/src/main/java/com/hazelcast/stabilizer/agent/workerjvm/WorkerJvmLauncher.java

```diff
@@ -34,8 +37,11 @@
     private final static File STABILIZER_HOME = getStablizerHome();
     private final static String CLASSPATH_SEPARATOR = System.getProperty("path.separator");
     private final static AtomicLong WORKER_ID_GENERATOR = new AtomicLong();
+    private final static String WORKERS_PATH = "/home/users/stabilizer/hazelcast-stabilizer-0.4-SNAPSHOT/workers";
```

Comment:
Why did you hard code this? You can read out system property to determine home directory and you already know the stabilizer_home, so workers you can derive from that.